### PR TITLE
Add net.base_art.Glide

### PR DESCRIPTION
**Pull request by philn**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly.
  
Glide is a simple and minimalistic media player relying on GStreamer for the multimedia support
and GTK for the user interface. Glide should be able to play any multimedia format supported by
GStreamer, locally or remotely hosted.

This is a new submission aiming to rename Glide and verify it using my domain name.

- [x] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read the and followed all the [Submission and licence requirements][reqs].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by philn** [comment]
Current name is net.baseart.Glide ... See https://flathub.org/apps/net.baseart.Glide

File: net.base_art.Glide.yaml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        dest-filename: autogen.sh
+  - name: ffmpeg
+    builddir: true
+    sources:
+      - type: git
+        url: https://git.ffmpeg.org/ffmpeg.git
+        tag: n6.1.1
+    config-opts:
+      - --prefix="/app/"
+      - --libdir="/app/lib"
```
Comment (bbhtt) [inline net.base_art.Glide.yaml:44]:
```suggestion
```

this is set automatically

File: 0001-Rename-app-to-net.base_art.Glide.patch [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+ src/ui_context.rs                              |  6 +++---
+ 14 files changed, 38 insertions(+), 32 deletions(-)
+ rename build-aux/{net.baseart.Glide.Devel.json => net.base_art.Glide.Devel.json} (84%)
+ rename data/{net.baseart.Glide-symbolic.svg => net.base_art.Glide-symbolic.svg} (100%)
+ rename data/{net.baseart.Glide.Devel.svg => net.base_art.Glide.Devel.svg} (100%)
+ rename data/{net.baseart.Glide.Source.svg => net.base_art.Glide.Source.svg} (99%)
+ rename data/{net.baseart.Glide.desktop => net.base_art.Glide.desktop} (99%)
+ rename data/{net.baseart.Glide.metainfo.xml => net.base_art.Glide.metainfo.xml} (95%)
+ rename data/{net.baseart.Glide.svg => net.base_art.Glide.svg} (100%)
+ rename data/{net.baseart.Glide.ui => net.base_art.Glide.ui} (100%)
```
Comment (bbhtt) [inline 0001-Rename-app-to-net.base_art.Glide.patch:30]: this patchfile seems to have a lot of unrelated and unnecessary changes.

You have a `flatpakref` in there.

Icons metainfo and desktop file can be renamed using `rename-icon, rename-desktop-file, rename-appdata-file`. see https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html

The rest should be trimmed a bit, for example, we don't need changes in the manifest in git or your github workflows.

**Comment by bbhtt** [comment]
bot, build net.base_art.Glide

File: 0001-Rename-app-to-net.base_art.Glide.patch (hunk 2 above)
Comment (philn) [reply 0001-Rename-app-to-net.base_art.Glide.patch:30]: Eventually I need the Devel flatpak renamed too, for consistency. So I'm not sure what's the point of splitting these changes away.

Comment (philn) [reply 0001-Rename-app-to-net.base_art.Glide.patch:30]: > Icons metainfo and desktop file can be renamed using rename-icon, rename-desktop-file, rename-appdata-file. see https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html

Yes sure, then I would end-up with a mix of files, some keeping the old name and some the new one? I'm not sure that's better :(

Comment (philn) [reply 0001-Rename-app-to-net.base_art.Glide.patch:30]: Alright, I can do that ;) I'll update this PR for 0.6.5 then.

**Comment by bbhtt** [comment]
bot, build net.base_art.Glide

**Pull request closed without merging**